client/start_restores.py

```diff
@@ -0,0 +1,14 @@
+import sys
+import boto3
+
+
+def post_messages(session, shoot_numbers):
+    sns = session.resource("sns")
+    topic = sns.Topic(f"arn:aws:sns:eu-west-1:760097843905:restore_shoots-production")
```

Comment:
No point in making the env configurable here, if only for testing?

Reply:
Could do, but there is no staging transfer throttle, which is the only way in which this matters.

If we want to test things going to staging we can do that in steps using restore.py and start_transfers.py locally.